libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Analytics/SFSDKSalesforceAnalyticsManager.m

```diff
@@ -113,32 +113,31 @@ - (instancetype) initWithUser:(SFUserAccount *) userAccount {
 
         // Reads the existing analytics policy and sets it upon initialization.
         [self readAnalyticsPolicy];
```

Comment:
Shouldn't this be replaced by `self.loggingEnabled = isLoggingEnabled;` for initialization?